media-proxy/src/api_server_tcp.cc

```diff
@@ -279,6 +280,12 @@ void RunTCPServer(ProxyContext* ctx)
         return;
     }
 
+    /* Workaround to allow media_proxy to listen on the same port after termination and starting again */
+    if (setsockopt(sock, SOL_SOCKET, SO_REUSEADDR, &enable, sizeof(int)) < 0) {
+        fprintf(stderr, "error: cannot set SO_REUSEADDR");
+        return;
+    }
+
```

Comment:
This should bee moved to separate PR i think, as it is out of scope.

Reply:
This should stay with this PR since it's critical to starting the FFmpeg MCM plugin test runs. I extended the scope of this PR by adding bugfixing.